contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -75,8 +75,8 @@ const (
 	defaultReconcileCooldown = 15 * time.Second
 	defaultFrameworkName     = "Kubernetes"
 
-	executorCPUs = mresource.CPUShares(0.25) // initial CPU allocated for executor
-	executorMem  = mresource.MegaBytes(64.0) // initial memory allocated for executor
+	ExecutorCPUs = mresource.CPUShares(0.01) // initial CPU allocated for executor, intentionally set to the minimum allow by Mesos: 0.01
```

Comment:
This is currently the only way to pass the https://github.com/mesosphere/kubernetes-mesos/issues/437 e2e test. Can you create a GH issue about customizing this value?